tsconfig.json

```diff
@@ -1,12 +1,16 @@
 {
   "compilerOptions": {
-    "module": "commonjs",
+    "module": "esnext",
```

Comment:
this _might_ bite us with electron. but I hope not.

Reply:
I reverted this. Was leftover from debugging the compilation issue.